src/components/StageSelect.js

```diff
@@ -11,9 +12,39 @@ const animatedComponents = makeAnimated();
 export class StageSelect extends React.Component {
   constructor(props) {
     super(props);
+    this.state ={
+      flag : false
+    }
+  }
+
+  ConnectMerakiApi = () => {
+    axios.post(`https://connect.merakilearn.org/api/offerLetter/admissions`,{
+      "receiverEmail": "poonam@navgurukul.org",
+      "name": "Poonam",
+      "campus": "",
+      "cc": "anand@navgurukul.org saquib@navgurukul.org"
```

Comment:
@Poonam-Singh-Bagh Partner's email should be CC'ed. In case partner's email is empty, send empty strings in CC